src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-sentry_sdk.py

```diff
@@ -18,3 +20,21 @@
                  "sentry_sdk.integrations.argv",
                  "sentry_sdk.integrations.logging",
                  "sentry_sdk.integrations.threading"]
+
+statement = """
+import json
+import sentry_sdk.integrations as si
+
+integrations = []
+if hasattr(si, '_AUTO_ENABLING_INTEGRATIONS'):
+    # _AUTO_ENABLING_INTEGRATIONS is a list of strings with default enabled integrations 
```

Comment:
I think you're confusing *trailing* whitespace with *leading* whitespace (a.k.a indentation). All the linter is moaning about is (as @rokm says):
```suggestion
    # _AUTO_ENABLING_INTEGRATIONS is a list of strings with default enabled integrations
```